llpc/unittests/standaloneCompiler/testInputUtils.cpp

```diff
@@ -205,6 +205,41 @@ TEST(InputUtilsTest, IsSpirvBinaryFile) {
 
 TEST(InputUtilsTest, IsGlslShaderFile) {
   // Based on https://www.khronos.org/opengles/sdk/tools/Reference-Compiler/.
+  //
+  //Copyright (C) 2002-2005 3Dlabs Inc. Ltd.
+  //Copyright (C) 2012-2013 LunarG, Inc.
```

Comment:
This is not necessary IMO. The only thing based on the website is the list of extensions. The implementation is not based on the website.